### DISCUSSION TITLE
How to create a custom Markdown rules

### DISCUSSION THREAD

**Discussion by ArvindhMangoleap** [markedjs/marked#2543] [Q&A] [answered]
I am trying to build a parser that converts markdown into WhatsApp compatible format

For example,

Whatsapp uses underscore ( _ ) on both sides of the string for Italic instead of start ( * )

WhatsApp  ` _Italic_`
Regular Markdown ` *Italic*`

WhatsApp  `*Bold*`
Regular Markdown `**Bold**`

I can use custom extensions to overwrite bold, italics etc. 

Is there a way to disable all other render methods like list, paragraph, table, etc? Because I don't need them.

**Comment by UziTech** [comment, marked as answer]
you can create your own [tokenizer](https://marked.js.org/using_pro#tokenizer) and return undefined to prevent marked from creating a token.

```js
const tokenizer = {
  heading() {
    // return nothing/undefined to prevent tokenization
  }
};

marked.use({ tokenizer });
```
